piwik/etc/confd/templates/virtual-host.conf.tmpl

```diff
@@ -0,0 +1,17 @@
+<VirtualHost *:80>
```

Comment:
it isn't needed, and shouldn't be used. There doesn't seem to be a reason for adding the /server-status, but if there is, then please read https://github.com/continuouspipe/dockerfiles/tree/master/php-apache#addingreplacing-apache-httpd-configuration.

Piwik, like Google Analytics does, should be used with HTTPS only, so inheriting the base image template would enable that